adflow/pyADflow.py

```diff
@@ -569,8 +569,9 @@ def addSlices(self, direction, positions, sliceType="relative", groupName=None):
                 sliceName = "Slice_%4.4d %s Para Init %s=%7.3f" % (j, groupTag, direction, positions[i])
                 self.adflow.tecplotio.addparaslice(sliceName, tmp[i], dirVec, famList)
             else:
+                # e.g. Slice_0008 allWalls: Aboslute y=66.875
```

Comment:
This should definitely be removed. Any reason why `self.adflow.tecplotio.addabsslice` was removed here? @Aagaard87